build/Targets/Signing.Imports.targets

```diff
@@ -54,6 +54,8 @@
         <!-- Non-shipping binaries are simply signed with the Roslyn internal key. -->
         <Otherwise>
           <PropertyGroup>
+            <!-- Public sign on non-Windows as full signing isn't currently supported (https://github.com/dotnet/roslyn/issues/8210) -->
```

Comment:
We only set public-sign/delay-sign for non-shipping != true. Here we assume since we're using the internal key we can just full sign, but we can't.